### DISCUSSION TITLE
Save location of default generation settings?

### DISCUSSION THREAD

**Discussion by ItsGifNotJif** [JarodMica/ai-voice-cloning#135] [Q&A]
Anyone know where default settings for generation are saved? I'd like to adjust some defaults to load up the way I want every time without having to re-adjust with every launch of TTS. I've tried searching for default settings file to edit, but I probably missed it somewhere.

**Comment by GregMuck99** [comment]
The HISTORY tab will contain the settings you used to generate each of your audio results (located in the results folder). I always forget the exact sequence of 'clicks' I use to make the spreadsheet show up. I also know of no way to automatically load those setting into the generate tab, but the setting are there for retrieval.
